amplipi/app.py

```diff
@@ -81,7 +81,17 @@
 
 app.add_exception_handler(NotAuthenticatedException, not_authenticated_exception_handler)
 
-logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
should probably match what we already use for now - DEBUG.

Reply:
I've gotten the sense that DEBUG is meant to be used as needed and INFO is meant to be the default lowest level, but I'll totally follow your lead here